extensions/amp-audioburst-audio/0.1/test/validator-amp-audioburst-audio.protoascii

```diff
@@ -0,0 +1,43 @@
+#
```

Comment:
You don't need to put a protoascii file in the test directory, so this can be deleted.

Reply:
done